langchain/scripts/create-entrypoints.js

```diff
@@ -78,6 +78,7 @@ const entrypoints = {
   "vectorstores/faiss": "vectorstores/faiss",
   "vectorstores/weaviate": "vectorstores/weaviate",
   "vectorstores/lancedb": "vectorstores/lancedb",
+  "vectorstores/momento-vector-index": "vectorstores/momento-vector-index",
```

Comment:
Do you mind making this snake case instead of kebab to match the other entrypoints? So:

`"vectorstores/momento_vector_index"`

You'll need to rename the file as well

Reply:
Will do.

Reply:
@jacoblee93 Fixed in https://github.com/langchain-ai/langchainjs/pull/2851/commits/7615573f3cad748a47ba5398371db15f033cfb80 🖖